### PR TITLE
Ops 1198 bell icon

### PR DESCRIPTION
**Pull request by johndeange**
## What changed

Use different icon depending on if there are notifications.

## Issue

https://github.com/HHS/OPRE-OPS/issues/1198

## How to test

1. Log into the app and click the bell to see your notifications.
2. Clear the notifications.
3. The bell icon should change.

## Screenshots

image

image

### CODE REVIEW

**Review by tdonaworth — APPROVED** [review]

File: frontend/src/components/Auth/AuthSection.jsx [hunk 1]
```diff
@@ -106,8 +106,12 @@ const AuthSection = () => {
             {isLoggedIn && (
                 <div>
                     <div className="display-flex flex-align-center">
-                        <User />
-                        <NotificationCenter />
+                        <div style={{ paddingRight: "20px" }}>
```
Comment (fpigeonjr) [inline frontend/src/components/Auth/AuthSection.jsx:109]: pls use uswds spacing units `padding-right-205`

File: frontend/src/components/Auth/AuthSection.jsx [hunk 2]
```diff
@@ -106,8 +106,12 @@ const AuthSection = () => {
...
+                            <User />
+                        </div>
+                        <div style={{ paddingRight: "20px" }}>
```
Comment (fpigeonjr) [inline frontend/src/components/Auth/AuthSection.jsx:112]: pls use uswds spacing units `padding-right-205`

**Review by fpigeonjr — CHANGES_REQUESTED** [review]
minor css improvements

**Review by fpigeonjr — APPROVED** [review]
👍🏿

**Pull request merged**